src/lib_ccx/params.c

```diff
@@ -2299,39 +2338,26 @@ int parse_parameters(struct ccx_s_options *opt, int argc, char *argv[])
 			opt->usepicorder = 1;
 			continue;
 		}
-		if (strstr(argv[i], "-myth") != NULL)
+		if (strstr(argv[i], "-myth") != NULL || strstr(argv[i], "--myth") != NULL)
 		{
 			opt->auto_myth = 1;
 			continue;
 		}
-		if (strstr(argv[i], "-nomyth") != NULL)
+		if (strstr(argv[i], "-nomyth") != NULL || strstr(argv[i], "--nomyth") != NULL)
 		{
 			opt->auto_myth = 0;
 			continue;
 		}
-		if (strstr(argv[i], "-wtvconvertfix") != NULL)
+		if (strstr(argv[i], "-wtvconvertfix") != NULL || strstr(argv[i], "--wtvconvertfix") != NULL)
 		{
 			opt->wtvconvertfix = 1;
 			continue;
 		}
-		if (strstr(argv[i], "-wtvmpeg2") != NULL)
+		if (strstr(argv[i], "-wtvmpeg2") != NULL || strstr(argv[i], "--wtvmpeg2") != NULL)
 		{
 			opt->wtvmpeg2 = 1;
 			continue;
 		}
-		if (strcmp(argv[i], "-o") == 0)
```

Comment:
Why?

Reply:
This (-o and --outputfilename) was moved to line 1702 above --out because --outputfilename was being parsed as --out.

Reply:
Got it